statusboard/src/components/ProjectsTable/SuperResponsiveTableStyle.css

```diff
@@ -0,0 +1,85 @@
+/* inspired by: https://css-tricks.com/responsive-data-tables/ */
+.responsiveTable {
+  width: 100%;
+}
+
+.responsiveTable td .tdBefore {
+  display: none;
+}
+
+.mobile {
+  display:none;
+}
+
+.mobileheader {
+  display:none;
+}
+
+@media screen and (max-width: 800px) {
+  .desktop {
+    display:none;
+  }
+  .mobile {
+    display:block;
+  }
+
+  .mobileheader {
+    display:block;
+  }
+
+  /*
+    Force table elements to not behave like tables anymore
+    Hide table headers (but not display: none;, for accessibility)
+  */
+
+  .responsiveTable table,
+  .responsiveTable thead,
+  .responsiveTable tbody,
+  .responsiveTable th,
+  .responsiveTable td,
+  .responsiveTable tr {
+    display: block;
+  }
+
+  /* .responsiveTable thead tr {
+    position: absolute;
+    top: -9999px;
+    left: -9999px;
+    border-bottom: 2px solid #333;
+  } */
```

Comment:
I'd suggest not leaving the commented-out code in. Perhaps add a link on the top of the file with a link to the responsive table documentation?

Reply:
Done!